packages/Benchmarks/HybridTransport/src/CMakeLists.txt

```diff
@@ -29,6 +19,7 @@ TRIBITS_ADD_LIBRARY(
   dtk_hybridtransport
   HEADERS ${HEADERS}
   SOURCES ${SOURCES}
+  DEPLIBS dtk_utils
```

Comment:
I guess we have to register these dependency by hands now :/

Reply:
Yes, we need to establish the order of compiling, and `DEPLIBS` is the way to go in TriBITS. It is possible that I missed some of them.